src/components/DragAndDrop/Provider/types.ts

```diff
@@ -8,7 +8,7 @@ type DragAndDropProviderProps = {
     isDisabled?: boolean;
 
     /** Indicate that users are dragging file or not */
-    setIsDraggingOver: (value: boolean) => void;
+    setIsDraggingOver?: (value: boolean) => void;
```

Comment:
Are you sure this should be optional?

Reply:
Yes, @VickyStash explained the reason here: https://github.com/Expensify/App/pull/37820#discussion_r1520094999